packages/connect-node/src/connect-http2-transport.ts

```diff
@@ -56,7 +58,21 @@ import {
   readToEnd,
   write,
 } from "./private/io.js";
-import { connectErrorFromNodeReason } from "./private/connect-error-from-node.js";
+import {
+  connectErrorFromNodeReason,
+  getNodeErrorProps,
+} from "./private/node-error.js";
+import type { Compression } from "./compression.js";
+import { compressionBrotli, compressionGzip } from "./compression.js";
+import { validateReadMaxBytesOption } from "./private/validate-read-max-bytes-option.js";
+
+const messageFlag = 0b00000000;
+/**
+ * compressedFlag indicates that the data in a EnvelopedMessage is
+ * compressed. It has the same meaning in the gRPC-Web, gRPC-HTTP2,
+ * and Connect protocols.
+ */
+const compressedFlag = 0b00000001;
```

Comment:
since we will probably use this flag in the other transports, does it make sense to export this from the compression lib so we can import it instead of redeclaring it in the other transports?